frontend/src/components/auth/GroupMemberships.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <ul class="list-group">
+    <li class="list-group-item" v-for="group in memberships" :key="group.id">
+      {{ group.name }}
+    </li>
+    <li v-if="memberships.length === 0" class="list-group-item p-0">
+      <div class="alert alert-info text-center m-0">
+        Not a member of any group.
+      </div>
+    </li>
+  </ul>
+</template>
+<script setup>
+import { api } from "@/composables/api";
+import { onActivated } from "vue";
+
+const props = defineProps(["username"]);
+const memberships = await api.getUserMemberships({ username: props.username });
+
+onActivated(async () => {
+  memberships.value = await api.getUserMemberships({ username: props.username });
+})
```

Comment:
Yes, I missed that one. Indeed, since the keep-alive is removed now it doesn't make sense to do this anymore - the component will always be recreated.